### PR TITLE
Misc wording and grammar fixes in chapter 6

### PR DESCRIPTION
**Pull request by rwestlund**

### CODE REVIEW

File: chap6.tex [hunk 1]
```diff
@@ -9,14 +9,14 @@
...
-chapter, we take advantage from the fact that we gradually gain more
+One of the most important questions we would like to answer is \emph{given an
+initial distribution of scores, where should we be sampling from?} In this
+chapter, we take advantage of the fact that we gradually gain more
 information about the score distribution over time to dynamically determine
 where we should be sampling from.
 
-First, we perform different statistical analyses on the data. Then we propose a
-new sampling scheme based on our analyses. Finally, we set up an experiment to
+First, we perform different statistical analysis on the data. Then we propose a
```
Comment (vanallen) [inline chap6.tex:18]: even though we are having multiple analyses?

Comment (rwestlund) [reply chap6.tex:18]: You're right, I didn't do this properly. See #39 for a fix.

Comment (rwestlund) [reply chap6.tex:18]: Also, you don't have to merge pull requests if you see a mistake; I could have updated it, rather than making a new one.

Comment (vanallen) [reply chap6.tex:18]: ok

**Pull request merged**